tests/conftest.py

```diff
@@ -45,13 +45,17 @@ def __init__(self, method, base_url, url, params, json):
             self._url = url
             self._json = json
 
+            # Extract the ID from the URL if it's a specific resource request
+            self._id = None
+            if "/" in url and not url.endswith("s"):
```

Comment:
I don't quite get this condition "ending with s"

Reply:
If the url ends in `s`, then it's a resource and it doesn't contain an id